rules-emerging-threats/2023/Exploits/CVE-2023-46214/web_cve_2023_46214_shell_upload.yml

```diff
@@ -0,0 +1,31 @@
+title: Detection of CVE-2023-46214 POC Upload Of Shell
+id: ba5268de-4dd4-4d5c-8a90-2b5e6dc1aff8
+status: experimental
+description: This rule is based on the POC by nathan31337, and search for explotation of CVE-2023-46214
+references:
+    - https://github.com/nathan31337/Splunk-RCE-poc/tree/main
+    - https://nvd.nist.gov/vuln/detail/CVE-2023-46214
+author: Lars B. P. Frydenskov(Trifork Security)
+date: 2023/11/19
+tags:
+    - cve.2023.46214
+    - detection.emerging_threats
+    - attack.lateral_movement
+    - attack.t1210
+logsource:
+    category: webserver
+detection:
+    selection_method:
+        cs-method: POST
+    selection_uris:
+        cs-uri-query|contains:
+            - /splunkd/__upload/indexing/preview\?output_mode=json\&props.NO_BINARY_CHECK=1&input.path=shell.xsl
+            - /api/search/jobs/[0-9]*/results\?xsl=/opt/splunk/var/run/splunk/dispatch/.*/shell.xsl
```

Comment:
Hi,
It is a regex not a string search.
Should be like :
```yaml
cs-uri-query|contains|all:
    - '/api/search/jobs/'
    - '/results\?xsl=/opt/splunk/var/run/splunk/dispatch/'
    - '/shell.xsl'
```

Reply:
I see, thanks for pointing it out